larky/src/main/resources/stdlib/urllib/request.star

```diff
@@ -1,214 +1,294 @@
-# Request -- An object that encapsulates the state of a request.  The
-# state can be as simple as the URL.  It can also include extra HTTP
-# headers, e.g. a User-Agent.
+"""An extensible library for opening URLs using a variety of protocols
```

Comment:
spacing in this entire doctstring is off, compare with: https://raw.githubusercontent.com/python/cpython/3.10/Lib/urllib/request.py